src/Theme/HeaderTemplate.elm

```diff
@@ -86,3 +91,54 @@ headerBrandStyle =
         , fontFamilies [ "Ludicrous" ]
         , margin zero
         ]
+
+
+headerContainerStyle : Style
+headerContainerStyle =
+    batch
+        [ alignItems baseline
+        , centerContent
+        , displayFlex
+        , flexDirection column
+        , flexWrap noWrap
+        , justifyContent center
+        , outerPadding
+        , withMediaMobileUp
+            [ flexDirection row
+            , justifyContent spaceBetween
+            ]
+        ]
+
+
+searchButtonsContainerStyle : Style
+searchButtonsContainerStyle =
+    batch
+        [ alignItems flexStart
+        , displayFlex
+        , flexDirection column
+        , flexWrap noWrap
+        , justifyContent center
+        , marginTop (rem 2)
+        , marginBottom (rem 0)
+        , withMediaMobileUp
+            [ margin2 (rem 2) (rem 0)
+            , alignItems flexEnd
+            ]
+        ]
+
+
+headerLinkStyle : Style
+headerLinkStyle =
+    batch
+        [ textAlign left
+        , withMediaMobileUp
+            [ textAlign right
+            ]
+        ]
+
+
+headerTitleStyle : Style
+headerTitleStyle =
+    batch
+        [ marginRight (rem 3)
```

Comment:
should this be `withMediaMobileUp` 

Might want to consider enforcing the wrapping at larger sizes but could also be out of scope for this ticket

Reply:
yes it should!